hack/install-e2e-dependencies.sh

```diff
@@ -0,0 +1,25 @@
+set -eux
+curl -Lo helm.tar.gz \
```

Comment:
doesn't need to be fixed in this PR - we should have a consistent header for bash files/files in general, and have it verified by CI that all files are up to date. Not one for this PR though, we don't have any 'enforced' style at the moment.